Automattic/wp-calypso
# Generated design picker: Generated design endpoint migration

#### Changes proposed in this Pull Request
This PR changes the endpoint used to fetch generated designs to `starter-designs/generated` endpoint. This new endpoint will handle all the logic that goes into determining which designs are shown to users in the generated design picker, thus reducing complexity on client-side.

This PR depends on D81174-code

#### Testing instructions



* Head to `/setup/vertical?siteSlug=${site_slug}`, set a vertical (I recommend Arts & Entertainment)
* Head to `/setup/intent?siteSlug=${site_slug}` and open up your browser's developer tools.
* Click on the "Start building" button, and you should land on the generated design picker screen.
* Check the developer tools' Network tab and confirm that the endpoint `starter-designs/generated` is called.
* Expect the designs returned by the endpoint to match with the list of generated designs on the left side thumbnail.
* Expect that iframe renders works as before.
* Expect that design selection and submission works as before.
